modules/apps/callbacks/types/callbacks_test.go

```diff
@@ -278,8 +277,7 @@ func (s *CallbacksTypesTestSuite) TestGetCallbackData() {
 
 			tc.malleate()
 
-			testPacket := channeltypes.Packet{Data: packetData}
-			callbackData, err := types.GetCallbackData(packetDataUnmarshaler, testPacket, remainingGas, uint64(1_000_000), callbackKey)
+			callbackData, err := types.GetCallbackData(packetDataUnmarshaler, packetData, "", remainingGas, uint64(1_000_000), callbackKey)
```

Comment:
could use `ibcmock.PortID` instead of empty string?

ref: https://github.com/cosmos/ibc-go/blob/main/testing/mock/mock.go#L30